third_party/move/move-bytecode-verifier/src/stack_usage_verifier.rs

```diff
@@ -228,6 +228,43 @@ impl<'a> StackUsageVerifier<'a> {
                 (arg_count, return_count)
             },
 
+            // ClosEval pops the number of arguments and pushes the results of the given function
+            // type
+            Bytecode::CallClosure(idx) => {
+                if let Some(SignatureToken::Function(args, result, _)) =
+                    self.resolver.signature_at(*idx).0.first()
+                {
+                    ((1 + args.len()) as u64, result.len() as u64)
+                } else {
+                    // We don't know what it will pop/push, but the signature checker
+                    // ensures we never reach this
+                    (0, 0)
+                }
+            },
+
+            // ClosPack pops the captured arguments and returns 1 value
+            Bytecode::PackClosure(idx, mask) => {
+                let function_handle = self.resolver.function_handle_at(*idx);
+                let arg_count = mask
+                    .extract(
```

Comment:
1. Why extract, we can probably use `popcount` to count number of 1s in the mask?
2. We need to check that arg_count <= 64 somewhere